support-workers/src/test/scala/com/gu/acquisitions/BigQuerySpec.scala

```diff
@@ -77,7 +77,7 @@ class BigQuerySpec extends AsyncFlatSpec with Matchers with LazyLogging {
       None
     )
 
-    service.tableInsertRow(dataRow) shouldBe Right(())
+    service.tableInsertRow(dataRow).value.map(_ shouldBe Right(()))
```

Comment:
sorry I should have pointed out, this test was failing because Right(()) was not the same as Future(Not completed yet)